packages/react-core/src/components/Tabs/TabAction.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Tabs/tabs';
+import { Button } from '../Button';
+import { getOUIAProps, OUIAProps } from '../../helpers';
+
+export interface TabActionProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'ref' | 'type'>, OUIAProps {
+  /** Content rendered in the tab action */
+  children?: React.ReactNode;
+  /** Additional classes added to the tab action span */
+  className?: string;
+  /** Click callback for tab action button */
+  onClick?: (event: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+  /** Flag indicating if the tab action is a help action */
+  isHelpAction?: boolean;
+  /** Flag indicating if the tab action is disabled */
+  isDisabled?: boolean;
+  /** Accessible label for the tab action */
+  'aria-label'?: string;
+  /** @hide Callback for the section ref */
+  innerRef?: React.Ref<any>;
+}
+
+const TabActionBase: React.FunctionComponent<TabActionProps> = ({
+  children,
+  onClick,
```

Comment:
Can we destructure the `className` prop  and add it to the the className in the tabItemaction span on line 36.